backend/backend.go

```diff
@@ -186,6 +186,20 @@ func NewBackend(arguments *arguments.Arguments, environment Environment) (*Backe
 	backend.ratesUpdater = rates.NewRateUpdater(hclient)
 	backend.ratesUpdater.Observe(backend.Notify)
 	backend.ratesUpdater.Start()
+	for _, fiat := range config.AppConfig().Backend.FiatList {
+		if config.AppConfig().Backend.BitcoinActive {
+			backend.ratesUpdater.EnableHistoryPair("BTC", fiat)
+		}
+		if config.AppConfig().Backend.LitecoinActive {
+			backend.ratesUpdater.EnableHistoryPair("LTC", fiat)
+		}
+		if config.AppConfig().Backend.EthereumActive {
+			backend.ratesUpdater.EnableHistoryPair("ETH", fiat)
+		}
+		for _, token := range config.AppConfig().Backend.ETH.ActiveERC20Tokens {
+			backend.ratesUpdater.EnableHistoryPair(token, fiat)
```

Comment:
for sai and dai, this will pass `sai0x89d2` and `dai0x6b17`, but in rates.go they are `SAI` and `DAI`.

probably better to adjust the coin code for those in rates.go instead.

Reply:
Oh, I didn't realize these token codes are different between rates.go and the frontend. I can see the map in settings.tsx: https://github.com/digitalbitbox/bitbox-wallet-app/blob/eeff139/frontends/web/src/routes/settings/settings.tsx#L62-L71.  Let me see... but yeah, maybe using `coin.Code` is better.